### PR TITLE
Allow configuration of Altair UI settings

### PR DESCRIPTION
**Pull request by Sonic198**
Add support for Altair UI settings

### CODE REVIEW

**Comment by Shane32** [comment]
Looks good. You’ll need to run the api tests so that the approval file is updated and CI tests pass. 

I assume you tested this PR and the changes work sufficiently for your needs?

**Comment by Sonic198** [comment]
you 

> Looks good. You’ll need to run the api tests so that the approval file is updated and CI tests pass.
> 
> I assume you tested this PR and the changes work sufficiently for your needs?

You mean I need to run tests locally from VS?
Yes, I tested it, works well for me.

**Comment by Shane32** [comment]
> You mean I need to run tests locally from VS?

Yes. It will update an approval file; just commit the changed file and push.

File: src/Ui.Altair/Internal/altair.cshtml [hunk 1]
```diff
@@ -61,6 +61,7 @@
         subscriptionsEndpoint: getSubscriptionsEndPoint(),
         initialHeaders: @Model.Headers,
         initialSubscriptionsPayload: @Model.SubscriptionsPayload,
+        initialSettings: @Model.Settings
```
Comment (Shane32) [inline src/Ui.Altair/Internal/altair.cshtml:64]:
```suggestion
        initialSettings: @Model.Settings,
```

**Review by Shane32 — APPROVED** [review]

**Comment by Shane32** [comment]
@sungam3r Do you want to review?  I'll merge/release tomorrow otherwise.

**Pull request merged**